tsconfig.app.json

```
@@ -21,7 +22,15 @@
     "noUnusedParameters": true,
     "erasableSyntaxOnly": true,
     "noFallthroughCasesInSwitch": true,
-    "noUncheckedSideEffectImports": true
+    "noUncheckedSideEffectImports": true,
+
+    /* Aliases */
+    "baseUrl": ".",
+    "paths": {
+      "@/*": ["src/*"],
+      "@components/*": ["src/components/*"],
+      "@pages/*": ["src/pages/*"]
+    }
```

аналогично тогда тут:
```suggestion
    "baseUrl": ".",
    "baseUrl": ".",
    "paths": {
      "@/*": ["src/*"]
    }
```